tests/test_resource.py

```diff
@@ -125,6 +150,17 @@ def test_accept_headers(self):
                           params=json.dumps({'test': 'yeah'})).json,
             {'test': 'yeah'})
 
+    def test_accept_headers_callable(self):
+        self.assertEqual(
+            self.app.post("/fruits", headers={'Accept': 'text/json'},
+                          params=json.dumps({'test': 'yeah'})).json,
+            {'test': 'yeah'})
+
+        self.assertEqual(
+            self.app.post("/fruits", headers={'Accept': 'application/json'},
+                          params=json.dumps({'test': 'yeah'})).json,
+            {'test': 'yeah'})
```

Comment:
We may want to add a call that fails with a 415 to make sure we actually use what `_accepts()` returns